stock_no_negative/models/stock_quant.py

```diff
@@ -39,7 +39,7 @@ def check_negative_qty(self):
             ):
                 msg_add = ""
                 if quant.lot_id:
-                    msg_add = _(" lot {}").format(quant.lot_id.name_get()[0][1])
+                    msg_add = _(" lot {}").format(quant.lot_id.display_name)
```

Comment:
Could you remove also `format()` method and use translation function arguments instead ?

```suggestion
                    msg_add = _(" lot %(name)s", name=quant.lot_id.display_name)
```

And add a test to avoid regression ? Thanks

Reply:
Hello, i changed the format function.
How should i create the test ? Should i assert the FULL ValidationError message content ?

the msg_add is used in a dynamic string passed to a validation error